ticket-booker/src/components/Navbar.style.js

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const NavbarContainer = styled.nav`
+  width: 95%;
+  height: 80px;
+  background-color: #222222;
```

Comment:
#222222 happens in 2 places.
Maybe you can create a constant for DRY

Reply:
created a colors.js file which has all the colors that can be imported and reused through the app.